### PR TITLE
Fix lint errors for API

### PR DESCRIPTION
**Pull request by alandoni**
I didn't change anything from integrations because I don't know how that works exactly and it doesn't have any test

### CODE REVIEW

**Comment by alandoni** [comment]
All API tests passing

File: api/integrations/sheetsIntegration.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ function fetchDatesFromResult(values, arrayMatches, index, yearToSave, monthToSa
     has to be made by the number of player's collumns and not the one from date */
     for (let dateIndex = 0; dateIndex < values[index + 1].length; dateIndex += 2) {
         // Date wil be null/undefined if it's the same as previous one
-        if ((values[index][dateIndex] != '') && (values[index][dateIndex] != undefined)) {
+        if ((values[index][dateIndex] != '') && (values[index][dateIndex] !== null)) {
```
Comment (kiosia) [inline api/integrations/sheetsIntegration.js]: change to `!=` so undefined also returns true :)

File: api/routes.js [hunk 2]
```diff
@@ -147,12 +147,12 @@ exports.set = function(app, mongo) {
 
     app.get(URL_SALT, (request, response) => {
         adminController.getByCriteria({ nickname : request.params.nickname }).then((users) => {
-            let user = users[0];
+            let user = users;
```
Comment (kiosia) [inline api/routes.js:150]: is there a reason for not using `users` directly? :thinking:

File: api/server.js [hunk 3]
```diff
@@ -4,17 +4,44 @@ const express = require('express');
 const MongooseController = require('./mongooseController');
 const path = require('path');
 const routes = require('./routes');
+const winston = require('winston');
+
```
Comment (kiosia) [inline api/server.js:8]: GOOD ONE! =D

File: api/utils.js [hunk 4]
```diff
@@ -25,14 +25,20 @@ module.exports = {
...
-        date = sumHour(date, -3);
+        date = this.sumHour(date, -3);
         return date;
     },
 
     setTime : function(date, hour, minute) {
         return new Date(date.getFullYear(), date.getMonth(), date.getDate(), hour, minute, 0, 0);
     },
 
+    sumHour : function(date, numberOfHours) {
```
Comment (kiosia) [inline api/utils.js:36]: Nothing broke without this function, so do we really need it? :thinking:

**Review by kiosia — CHANGES_REQUESTED** [review]

File: api/integrations/sheetsIntegration.js (hunk 1 above)
Comment (alandoni) [reply api/integrations/sheetsIntegration.js]: ops

File: api/routes.js (hunk 2 above)
Comment (alandoni) [reply api/routes.js]: It's personal, but I don't like to use array everytime

Comment (alandoni) [reply api/routes.js:150]: it's personal, I don't like to use array every time

File: api/utils.js (hunk 4 above)
Comment (alandoni) [reply api/utils.js]: it breaks once a year hehe

Comment (alandoni) [reply api/utils.js:36]: it is used on convertToDate, I don't know how it doesn't break anything yet

**Review by kiosia — APPROVED** [review]

**Pull request merged**